package.json

```diff
@@ -192,13 +196,16 @@
 		"mockery": "2.1.0",
 		"nock": "10.0.6",
 		"node-wp-i18n": "1.2.3",
+		"npm-run-all": "4.1.5",
```

Comment:
Do we need to introduce a new dependency here, instead of just chaining the different actions?

Reply:
We can totally just chain them but it should be faster to run in CI in parallel. How much faster — I'd have to check. :-)

I wonder if it would make sense to add `php:lint` script to the same run? (That would be good for another PR anyway)

Reply:
Yeah, if it is faster we should probably start using it everywhere we can :)

Reply:
Yeah, if it is faster we should probably start using it everywhere we can :)

Reply:
Yeah, I'd like to see some exploration adding parallelism to build steps for instance.

Reply:
Noting that we started using `concurrently` for some of our tasks; maybe we could use it here too?

Reply:
Oh ye, totes! Feel free to change unless I get to it first.

Reply:
@jeherve switched to `concurrently`